### PR TITLE
feat(ui-components) : add list-view support + update ui-components & common services

### PR DESCRIPTION
**Pull request by Mboulianne**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### CODE REVIEW

File: client/app/veeam/veeam-detail.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
             </ul>
         </oui-message>
     </div>
-    <div class="veeam-content">
+    <div>
         <div ui-view="veeamContent"></div>
     </div>
```
Comment (JDownloader) [inline client/app/veeam/veeam-detail.html]: Is this «div» useful? (lines 13 and 15)

File: client/app/ui-components/accordion/accordion.component.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                        <h5 role="button" tabindex="0"
+                            class="oui-header_5 cui-accordion__header-title"
+                            data-ng-bind="$ctrl.title"
+                            data-ng-click="$ctrl.toggle()"></h5>
+                        <div class="cui-accordion__header-subtitle" data-ng-bind="$ctrl.text"></div>
+                    </div>
+                    <div class="cui-accordion__button-container">
+                        <cui-dropdown-menu data-ng-if="$ctrl.actions.length">
+                            <cui-dropdown-menu-button
+                                ng-include="'app/ui-components/icons/button-action.html'">
```
Comment (JDownloader) [inline client/app/ui-components/accordion/accordion.component.js]: data-ng-include

File: client/app/app.js [hunk 3]
```diff
@@ -1,79 +1,133 @@
...
+            .setPageSize(10)
+            .setExpandButtonTemplate(`
+                <i role="button"
+                    class="oui-icon oui-icon-chevron-right oui-table__expand-button"></i>
+            `)
+            .setSelectorTemplate(`<div class="oui-checkbox">
+                <input class="oui-checkbox__input"
+                  id="{{$name}}"
+                  type="checkbox"
+                  ng-model="$value"
```
Comment (JDownloader) [inline client/app/app.js]: data-ng-model/data-ng-change

File: client/app/app.js [hunk 4]
```diff
@@ -1,79 +1,133 @@
...
+            sortable: "oui-table__cell_sortable oui-table__cell_sortable-asc",
+            sorted: "oui-table__cell_sorted",
+            sortableAsc: "oui-table__cell_sortable-asc",
+            sortableDesc: "oui-table__cell_sortable-desc",
+            closed: "oui-table__row_closed",
+            emptyTable: "oui-table-empty"
+        })
+            .setPageSize(10)
+            .setExpandButtonTemplate(`
+                <i role="button"
```
Comment (JDownloader) [inline client/app/app.js]: not sure about this indent

Comment (Mboulianne) [reply client/app/app.js]: eslint expect an indent after a function call if chaining function calls.  The same apply for promises.

bla()
.then() => error.

bla()
    .then() => ok.

File: bower.json [hunk 5]
```diff
@@ -72,7 +72,10 @@
     "ovh-angular-doc-url": "^0.0.6",
     "ovh-angular-module-status": "^0.0.2",
     "ovh-angular-otrs": "^1.1.0",
-    "ovh-api-services": "^1.1.5"
+    "ovh-api-services": "^1.1.5",
+    "angular-chart.js": "^1.1.1",
+    "ovh-angular-list-view": "^0.1.1",
```
Comment (neolitec) [inline bower.json]: Try to use 0.1.5.

File: client/app/app.js [hunk 6]
```diff
@@ -1,79 +1,133 @@
...
+    .config(function ($urlRouterProvider, $locationProvider) {
+        "use strict";
+        $urlRouterProvider.otherwise("/");
+        $locationProvider.html5Mode(false);
+    })
+    .config(function (responsivePopoverProvider) {
+        // tell to the module that we consider a mobile device with at least 800px width
+        responsivePopoverProvider.setMobileMediaQuery("(max-width: 800px)");
+    })
+    .config(ChartJsProvider => {
```
Comment (neolitec) [inline client/app/app.js]: Lines 77, 78 and 79 should not be here. It's for ChartJS and I think it not even used in IPLB branch...

File: client/app/helper/order-helper.service.js [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+                return option;
+            });
+            formattedConfig.option = formattedOptions;
+        }
+
+        const paramsPart = this.$httpParamSerializerJQLike(_.assign({}, urlParams, {
+            products: JSURL.stringify([formattedConfig])
+        }));
+
+        console.log(paramsPart);
```
Comment (neolitec) [inline client/app/helper/order-helper.service.js]: ouch, sorry about that

File: client/app/helper/order-helper.service.spec.js [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+    beforeEach(module("managerAppMock"));
+
+    beforeEach(inject((_$q_, _$rootScope_, _$uibModal_, _OrderHelperService_) => {
+        $q = _$q_;
+        $rootScope = _$rootScope_;
+        OrderHelperService = _OrderHelperService_;
+    }));
+
+    describe("transformToOrderValues", () => {
+        fit("should accept simple values", () => {
```
Comment (neolitec) [inline client/app/helper/order-helper.service.spec.js]: If you merge you have to remove all `fit` in this file!
And than check if all tests pass.

File: bower.json (hunk 5 above)
Comment (Mboulianne) [reply bower.json]: list-view

**Pull request merged**